luda-editor/client/src/pages/sequence_edit_page/components/cut_editor/background_with_event.rs

```diff
@@ -99,6 +100,32 @@ impl Component for BackgroundWithEvent<'_> {
             }
         };
 
+        let handle_name_quick_slot_shortcut = |event: &KeyboardEvent| {
+            let ctrl_press = namui::keyboard::ctrl_press();
+            if !ctrl_press {
+                return;
+            }
+            let quick_slot_index = match event.code {
+                Code::Digit1 => 0,
+                Code::Digit2 => 1,
+                Code::Digit3 => 2,
+                Code::Digit4 => 3,
+                Code::Digit5 => 4,
+                _ => {
+                    return;
+                }
+            };
+            let name = name_quick_slot
+                .get_name(quick_slot_index)
+                .cloned()
+                .unwrap_or_default();
```

Comment:
is it demand to put empty string for empty slot?
I think showing notification that user didn't set up the slot and not change the character name of cut.

Reply:
Fixed 36795c9
![image](https://github.com/NamseEnt/namseent/assets/38313680/9a27a47a-05a0-4fe1-bfca-249bc5c612ef)